graphql-cli/build.gradle

```diff
@@ -105,13 +106,25 @@ jar {
     }
 }
 
+def disableGroups = ""
+
+task initializeVariables {
+    if (project.hasProperty("disable")) {
+        disableGroups = project.findProperty("disable")
+    }
+}
+
 test {
     dependsOn {
         copyStdlibs
+        initializeVariables
     }
     systemProperty "ballerina.home", bDistribution
 
     useTestNG() {
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            excludeGroups(disableGroups)
+        }
```

Comment:
This is not ideal, right? When you provide `disable` parameter, you'd expect them to be disabled in everywhere. Not just windows. The parameter should reflect that it is disabling the tests in Windows.

Reply:
Added the `disable` parameter with the purpose of decoupling the environment and the build script. This should disable the test everywhere. fixed in https://github.com/ballerina-platform/graphql-tools/pull/122/commits/f8dc8ae294fea4e703c351d8c99cb610856e78ee